databroker/_core.py

```diff
@@ -1102,6 +1102,13 @@ def __init__(self, hs, event_sources, assets, external_fetchers,
         self.registry_for_insert = self.event_sources[0]
         self.prepare_hook = wrap_in_deprecated_doct
 
+    @property
```

Comment:
The name `ES` strikes might as a bit cryptic. Why not `event_sources`, which is more self-describing and also symmetric with the name of the argument to `__init__`?

Reply:
No worries, it wasn't a personal criticism (doesn't matter who suggested it). Can we come up with a more  self-describing name? Is there a good reason why `event_sources` in `__init__` is a list and not a dict with names?